Samples/Samples/View/SMSPage.xaml

```diff
@@ -13,7 +13,7 @@
 
         <ScrollView>
             <StackLayout Padding="12,0,12,12" Spacing="6">
-                <Label Text="Recipent (optional):" />
+                <Label Text="Recipents (optional, separate multiple with .):" />
```

Comment:
I would change to ; instead of .

Reply:
This is not possible on Android. We use the keyboard for phone usage which does not have a ;

Reply:
ahhhhh gotcha! Thanks

Reply:
What about a `,`?

Reply:
Sure, we can change to a comma. It is more hidden thoug, only if you click on special signs does the comma become visible. I now changed it at your request.